src/fastoad/io/variable_io.py

```diff
@@ -11,34 +11,56 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import logging
 from fnmatch import fnmatchcase
-from os.path import exists, isfile
-from typing import IO, List, Sequence, Union
+from io import IOBase
+from os import PathLike
+from pathlib import Path
+from typing import List, Sequence, Union, IO, Optional
 
 from fastoad.openmdao.variables import VariableList
 from . import IVariableIOFormatter
 from .xml import VariableXmlStandardFormatter
+from .._utils.files import as_path
 from ..exceptions import FastError
 
+_LOGGER = logging.getLogger(__name__)  # Logger for this module
+
```

Comment:
The _LOGGER object does not seem to be used in the rest of the file, needs to be removed

Reply:
Fixed in a4b8a4bf354349003ee297fc5a891d1c7464f0f3